docs/src/data/hid.js

```diff
@@ -0,0 +1,5976 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+import { KEYBOARD, CONSUMER } from "./applications";
+
+export default [
+  {
+    names: ["A"],
+    description: "a and A",
+    context: "Keyboard",
+    clarify: false,
+    usages: [{ application: KEYBOARD, item: 458756 }],
```

Comment:
What is the `item` value here?

Reply:
That's the key/control's extended usage value `(page << 16) | id`.  The `application` of said key, is also an extended usage (`KEYBOARD` or `CONSUMER` at the moment, soon to be `MOUSE` as well).

It's how it's described in the HID specification, similar to the terminology we were using in #213.  I've yet to find a better alternative than `item` but I completely understand that it's not clear unless you know HID well.

Reply:
Also, `usages` is an array to facilitate simultaneous events.  Modified keys is the basic example of this.  But I foresee it being just as applicable to modified mice clicks, or shortcuts, etc.

Reply:
No worries. That makes sense, thanks for explaining it.

Reply:
When I get more time, I might replace those values with their `(page << 16 | id)` equivalents to increase readability.  It was originally JSON which explains why it's in that form currently.